extensions/wikia/TrackingOptIn/js/cmp.js

```diff
@@ -116,6 +116,39 @@ define('wikia.cmp', [
 		};
 	}
 
+	function getQuantcastLabels() {
+		var quantcastLabels = "";
+
+		if (window.wgWikiVertical) {
+			quantcastLabels += window.wgWikiVertical;
+
+			if (window.wgDartCustomKeyValues) {
+				var keyValues = window.wgDartCustomKeyValues.split(';');
+				for (var i=0; i<keyValues.length; i++) {
+					var keyValue = keyValues[i].split('=');
+					if (keyValue.length >= 2) {
+						quantcastLabels += ',' + window.wgWikiVertical + '.' + keyValue[1];
+					}
+				}
+			}
+		}
+
+		return quantcastLabels;
+	}
+
+	function loadQuantserveImage(optIn) {
```

Comment:
actually, you can move it here https://github.com/Wikia/app/blob/dev/extensions/wikia/AnalyticsEngine/AnalyticsProviderQuantServe.php#L31 and use window.__cmp() to retrieve consent data.